azext_hanaonazure/_params.py

```diff
@@ -11,4 +11,11 @@
 def load_arguments(self, _):
     with self.argument_context('hanainstance') as c:
         c.argument('resource_group_name', arg_type=resource_group_name_type)
-        c.argument('instance_name', options_list=['--instance-name', '-n'], help="The name of the SAP HANA instance", id_part='name')
\ No newline at end of file
+        c.argument('instance_name', options_list=['--instance-name', '-n'], help="The name of the SAP HANA instance", id_part='name')
+        c.argument('hana_vnet', options_list=['--hana-vnet', '-hv'], help="ARM ID of an Azure Vnet with access to the HANA instance.")
+        c.argument('hana_hostname', options_list=['--hana-hostname'], help="Hostname of the HANA Instance blade.")
+        c.argument('hana_instance_num', options_list=['--hana-instance-num'], help="A number between 00 and 99, stored as a string to maintain leading zero.")
+        c.argument('db_container', options_list=['--db-container'], help="Either single or multiple depending on the use of MDC(Multiple Database Containers). Possible values include: 'single','multiple'")
+        c.argument('hana_database', options_list=['--hana-database', '-hdb'], help="Name of the database itself.  It only needs to be specified if using MDC")
+        c.argument('hana_db_username', options_list=['--hana-db-username'], help="Username for the HANA database to login to for monitoring")
+        c.argument('hana_db_password', options_list=['--hana-db-password', '-hdbpw'], help="Password for the HANA database to login for monitoring")
```

Comment:
We might not want to let the user provide the password like this. It will show a password on their shell history.
We should have the user enter this in much like when you ssh, where the thing you type is not visible.
Or specify a file with the password in it.

Check to see if other commands to this. I am not sure how to do this off the top of my head.

Reply:
This is the same way that it is done for the virtual machine creation with a password https://github.com/Azure/azure-cli/blob/a32eb7156d7e814d38e2374e77fbd4d131bfe7f6/src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py. I agree with your point, but not sure how to adjust while still using the library

Reply:
-p is pretty standard... but I would say that if -p is not present, it the command should expect it from stdin, that way the user can 
cmd < /path/to/password_file